Makefile

```
@@ -474,11 +474,11 @@ delete-workload-cluster: ## Delete a CAPZ workload cluster.
 ##@ Tools
 
 LINTER = $(shell pwd)/bin/golangci-lint
-LINTER_VERSION = v2.5.0
+LINTER_VERSION = v2.9.0
 .PHONY: golangci-lint
 golangci-lint:  ## Download golangci-lint locally if necessary.
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)